src/gfn/losses/sub_trajectory_balance.py

```diff
@@ -73,8 +67,8 @@ def __init__(
     def cumulative_logprobs(
         self,
         trajectories: Trajectories,
-        log_p_trajectories: LogPTrajectoriesTensor,
-    ) -> LogPTrajectoriesTensor:
+        log_p_trajectories: TType["max_length", "n_trajectories", torch.float],
+    ) -> TType["max_length", "n_trajectories", torch.float]:
```

Comment:
Here's an example where I would claim readability is significantly worse.

Reply:
This looks ok to me. Note that as in the example below, we could replace `torch.float` with `float`